types/address/cdc.go

```diff
@@ -0,0 +1,9 @@
+package address
```

Comment:
not sure if this is the right package

Reply:
I personally find `address.Codec` okay. What pkg do you propose @aaronc ?